presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -225,10 +225,13 @@ Property Name                                           Description
 
 ``iceberg.enable-parquet-dereference-pushdown``         Enable parquet dereference pushdown.                          ``true``
 
-``iceberg.hive-statistics-merge-strategy``              Determines how to merge statistics that are stored in the     ``NONE``
-                                                        Hive Metastore. The available values are ``NONE``,
-                                                        ``USE_NULLS_FRACTION_AND_NDV``, ``USE_NULLS_FRACTIONS``
-                                                        and, ``USE_NDV``
+``iceberg.hive-statistics-merge-strategy``              Comma separated list of statistics to use from the
```

Comment:
If the behavior of this flag is changing from release to release, I think we'd want to introduce a new flag for this behavior, and deprecate the old flag.

Reply:
Is there a process for deprecation? I felt this was safe, because very few (if any) users consume these flags except for our internal benchmarks. I can add a release note for it.

Also, since Presto doesn't have a 1.X release, According to SemVer rules, this should be OK to change between releases.

Reply:
@ZacBlanco that is definitely not the case.  Presto is over a decade old, and people rely on features not breaking release to release.

I think in this case though, it's fine, because it hasn't made it to our public documentation yet.  Please add a release note.

Reply:
The release notes have been added to the PR description